code/style.css

```diff
@@ -1,4 +1,227 @@
-html {
-  background: #0025ff;
-  color: #fff;
+
+@import url('https://fonts.googleapis.com/css2?family=Amatic+SC&family=Montserrat:wght@100&family=Open+Sans+Condensed:wght@300;700&display=swap');
+
+/* Mobile  */
+
+body {
+  box-sizing: border-box;
+  margin:0%;
+  font-family: 'Montserrat', sans-serif;
+  background-color: #f2f2f0;
 }
+  
+/* <!-- Header  -->  */
+
+header {
+  height: 130px;  
+  display: flex;  
+  align-items: center;  
+  padding: 16px;
+  color:#d0c8b6;
+ } 
+
+nav a {
+   text-decoration: none;
+   padding: 20px;
+}
+
+.polaroid {
+  box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.19);
+   width: 280x
+}
+
+.headertext {
+  font-family: 'Amatic SC', cursive;
+  font-size: 50px;
+  margin-left: 40px;
+}
+
+.title {
+   display: flex;
+   font-family: 'Amatic SC', cursive;
+}
+  
+ .logo {  
+   width: 80px; 
+   border-radius: 50%; 
+   margin-left: 40px;
+ } 
+      
+ nav { 
+  display: none; 
+  font-size: 14px;
+} 
+
+.header-box {
+  display: flex;
+  margin: 0%;
+  justify-content: center;
+}
+
+.hover-underline-animation {
+  display:inline-block;
+  position: relative;
+  color: #f2f2f0;
+}
+
+.hover-underline-animation:after {
+  content: '';
+  position: absolute;
+  width: 100%;
+  transform: scaleX(0);
+  height: 1px;
+  bottom: 0;
+  left: 0;
+  background-color: #aeafa0;
+  transform-origin: bottom right;
+  transition: transform 0.25s ease-out;
+}
+
+.hover-underline-animation:hover:after {
+  transform: scaleX(1);
+  transform-origin: bottom left;
+}
+
+ /* <!-- The news  -->  */
+  
+.newsbox1 {
+  display: flex; 
+  flex-wrap: wrap;
+  justify-content: center;
+}
+
+.child1 {
+  margin-bottom: 20px;
+  display: flex;
+  justify-content: center;   
+}
+
+.child2{
+  display: flex;
+  justify-content: center;
+}
+    
+.a  {
+  text-decoration: none;
+}
+
+.text {
+  width: 80%;
+ }
+
+.image1{ 
+  margin: 10px;    
+}
+ 
+   /* <!-- The other news  -->  */
+
+.Newsbox2 {
+  display: flex; 
+  height: 100px; 
+  flex-direction: row;
+  flex-wrap: wrap;
+  justify-content: center;
+}
+  
+.text{
+  margin-top: 0px; 
+  margin-right: 20px;
+  margin-left: 25px; 
+  text-align: justify;
+} 
+  
+.footer-text  { 
+  font-size: 14px;  
+  margin-top: 5px; 
+  text-transform: uppercase; 
+  text-align: center; 
+  padding-bottom: 20px;
+} 
+  
+.date { 
+  text-align: center; 
+  margin-top: 5px; 
+  font-size: 10px;
+} 
+  
+.image{ 
+  width: 300px; 
+  margin: 10px;
+ } 
+  
+   
+  /* tablet*/ 
+ 
+  @media (min-width: 668px) { 
+  
+header { 
+  justify-content: center; 
+} 
```

Comment:
I suggest that more indentation is used here and for the rest of the code inside the media query selectors.